seed/models/analysis_runs.py

```diff
@@ -0,0 +1,38 @@
+
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from seed.models import (
+    Analysis,
+    Cycle,
+    Property,
+    PropertyState
+)
+
+
+class AnalysisRun(models.Model):
+
+    CREATING = 10
+    READY = 20
+    QUEUED = 30
+    RUNNING = 40
+    FAILED = 50
+    STOPPED = 60
+    COMPLETED = 70
```

Comment:
@macintoshpie I really like the idea of going with 10, 20, etc. future flexibility. Thanks for suggesting it!